package.json

```diff
@@ -19,7 +19,13 @@
     "tdd": "cross-env NODE_ENV=test mocha --compilers js:babel-register --watch --require ./test/setup-jsdom.js test/**/*_test.js",
     "test": "cross-env NODE_ENV=test mocha --compilers js:babel-register --require ./test/setup-jsdom.js test/**/*_test.js"
   },
-  "prettierOptions": "--jsx-bracket-same-line --trailing-comma es5 --semi --tab-width 2",
+  "prettier": {
+    "jsxBracketSameLine": true,
+    "trailingComma": "es5",
+    "useTabs": false,
```

Comment:
Some of those options unnecessary passing the default values, I just passed them here because they used to be in `cs-format`